e2e/suite1/specs/gauge.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { e2e } from '@grafana/e2e';
+
+e2e.scenario({
+  describeName: 'Gauge Panel',
+  itName: 'Gauge rendering e2e tests',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    // open Panel Tests - Gauge
+    e2e.flows.openDashboard({ uid: '_5rDmaQiz' });
+
+    cy.wait(1000);
+
+    // check that gauges are rendered
+    e2e()
+      .get('body')
+      .find(`.flot-base`)
```

Comment:
Wonder if we maybe could introduce label selector for Gauge viz rather than relying on this confusing class name? WDYT?

Reply:
not sure how without modifying the flot code, want to validate that the flot canvas exists